client.go

```diff
@@ -32,22 +32,22 @@ func NewClient(api plugin.API) *Client {
 	return &Client{
 		api: api,
 
-		Bot:           BotService{api},
-		Channel:       ChannelService{api},
-		Configuration: ConfigurationService{api},
-		SlashCommand:  SlashCommandService{api},
-		Emoji:         EmojiService{api},
-		File:          FileService{api},
-		Frontend:      FrontendService{api},
-		Group:         GroupService{api},
-		KV:            KVService{api},
-		Log:           LogService{api},
-		Mail:          MailService{api},
-		Plugin:        PluginService{api},
-		Post:          PostService{api},
-		Session:       SessionService{api},
-		System:        SystemService{api},
-		Team:          TeamService{api},
-		User:          UserService{api},
+		Bot:           BotService{api: api},
+		Channel:       ChannelService{api: api},
+		Configuration: ConfigurationService{api: api},
+		SlashCommand:  SlashCommandService{api: api},
+		Emoji:         EmojiService{api: api},
+		File:          FileService{api: api},
+		Frontend:      FrontendService{api: api},
+		Group:         GroupService{api: api},
+		KV:            KVService{api: api},
+		Log:           LogService{api: api},
+		Mail:          MailService{api: api},
+		Plugin:        PluginService{api: api},
+		Post:          PostService{api: api},
+		Session:       SessionService{api: api},
+		System:        SystemService{api: api},
```

Comment:
I did alter this to avoid having to pass `nil` for `*model.Manifest`

Reply:
Can we revert these changes now that we're not caching?

Reply:
We could, but I would like to keep them as they make it clear what is used to create the services.